dotcom-rendering/src/server/htmlPageTemplate.ts

```diff
@@ -322,6 +322,10 @@ https://workforus.theguardian.com/careers/product-engineering/
 					window.curl = window.curlConfig;
 				</script>
 
+				<script>
+					window.guardian.isSignedInAuthState = new Promise(r => window.guardian.isSignedInAuthStateResolve = r)
```

Comment:
What do you think about wrapping this up under an `identity` key?

```suggestion
					window.guardian.identity.isSignedInAuthState = new Promise(r => window.guardian.identity.isSignedInAuthStateResolve = r)
```

a bit like `ophan` and `readerRevenue` are doing